pymatviz/ptable.py

```diff
@@ -332,17 +334,23 @@ def __init__(
             colormap (str | Colormap | None): The colormap to use.
             plot_kwargs (dict): Additional keyword arguments to
                 pass to the plt.subplots function call.
+            hide_f_block: Hide f-block (Lanthanum and Actinium series).
         """
         # Get colormap
         self.cmap: Colormap = colormap
 
+        self.hide_f_block = hide_f_block
+
         # Preprocess data
         self.data: pd.DataFrame = data
 
         # Initialize periodic table canvas
         n_periods = df_ptable.row.max()
         n_groups = df_ptable.column.max()
 
+        if self.hide_f_block:
+            n_periods -= 3
```

Comment:
I removed the bottom three rows `if hide_f_block`, which not only enables unit test check for `len(axes)`, but also helped avoid the extra space at the bottom. 

@janosh. Do you have any further comments? Thanks!

Reply:
> just testing one of them should suffice.

i'd prefer to test multiple here